src/offline.js

```diff
@@ -136,6 +139,11 @@ export function replicateFromCozy (cozy, doctype, options = {}) {
       return reject(new Error('You can\'t use `live` option with Cozy couchdb.'))
     }
 
+    if (isOffline()) {
+      reject(replicationOfflineError)
+      return options.onError && options.onError(replicationOfflineError)
```

Comment:
I was wrong, it seems to be executed, but the return value is not what you think it would be: https://jsbin.com/xuyayot/1/edit?js,console